draft-yasskin-http-origin-signed-responses.md

```diff
@@ -959,6 +964,20 @@ extension. This OID might or might not be used as the final OID for the
 extension, so certificates including it might need to be reissued once the final
 RFC is published.
 
+### Extensions to the CAA Record: cansignhttpexchanges Parameter {#caa-cansignhttpexchanges}
+
+A CAA parameter "cansignhttpexchanges" is defined for the "issue" and
+"issuewild" properties defined by {{!RFC6844}}.  The value of this parameter, if
+specified, SHOULD be "yes".
+
+A CA SHOULD NOT issue certificates with the CanSignHttpExchanges extension
+defined in {{cross-origin-cert-req}} unless an applicable issue or issuewild
+property exists for the CA, and the "cansignhttpexchanges" parameter is present
+on the property and is equal to "yes".
+
+The CA/Browser Forum Baseline Requirements ({{BRs}}) are expected to establish
+requirements around the treatment of this parameter.
```

Comment:
1) In general, we should avoid SHOULD/SHOULD NOT requirements unless we provide guidance on when there are applicable exceptions. Alternatively, MUST/MUST NOT provides a clearer, unambiguous processing model, modulo what error handling should be (which presumably is MUST NOT issue)

2) The reference to the BRs may end up being circular. The BRs state the rules for which a CA can include arbitrary extensions (7.1.2.4.), so if there are requirements expressed here, they should trigger the rules of 7.1.2.4 (a) (ii) and 7.1.2.4 (b), and thus require following this spec and whatever it describes.

Reply:
I've moved this to the definition of the CanSignHttpExchanges *extension*, where there's no question that we can MUST CAs. Now I don't need to point at the BRs.